### PR TITLE
Translate Community/Tools/Misc page

### PR DESCRIPTION
**Pull request by linsolas**
Hello,

Ma traduction pour la page https://reactjs.org/community/misc.html

### CODE REVIEW

File: content/community/tools-misc.md [hunk 1]
```diff
@@ -1,19 +1,18 @@
...
-* **[react-render-visualizer](https://github.com/redsunsoft/react-render-visualizer):** A visual way to see what is (re)rendering and why.
-* **[javascript-monads](https://github.com/dschalk/javascript-monads):** As this project matures, the monads are becoming more than mere toys. React is used in unorthodox ways, so if you want to use React the way they do at Facebook, this is not for you. Transpiled code for the three monad classes can be downloaded by entering "npm install reactive-monads".
-* **[ReactCSS](http://reactcss.com/):** Inline Styles in JS with support for React.
-* **[ReactQuestionnaire](https://github.com/kouryuu/react-questionnaire):** Simple react components for building a questionnaire or survey.
-* **[renderjs.io](https://renderjs.io/):** SEO for Reactjs. Service for making reactjs application crawlable and shareable.
+* **[react-localize-redux](https://github.com/ryandrewjohnson/react-localize-redux)** : bibliothèque de localisation pour React / Redux.
+* **[react-inspector](https://github.com/xyc/react-inspector)** : utilisez des inspecteurs d'objets du type DevTools au sein de vos apps React.
```
Comment (sheplu) [inline content/community/tools-misc.md]:
```suggestion
* **[react-inspector](https://github.com/xyc/react-inspector)** : utilisez des inspecteurs d'objets du type DevTools au sein de vos applis React.
```
normalisation

Comment (tdd) [reply content/community/tools-misc.md]: :+1:

File: content/community/tools-misc.md [hunk 2]
```diff
@@ -1,19 +1,18 @@
...
+* **[jreact](https://github.com/KnisterPeter/jreact)** : réalise le rendu de React côté serveur au sein d'une JVM (Java 7 ou Java 8).
+* **[react-jss](https://github.com/jsstyles/react-jss)** : injecte et monte des styles JSS dans des composants React.
+* **[django-react](https://github.com/markfinger/django-react)** : rendu côté serveur de composants React pour des applications Django.
+* **[react-engine](https://github.com/paypal/react-engine)** : moteur de rendu composite pour des applications Express isomorphiques faisant le rendu de vues React et de vues react-router.
```
Comment (tdd) [inline content/community/tools-misc.md]:
```suggestion
* **[react-engine](https://github.com/paypal/react-engine)** : moteur de rendu composite pour des applications Express isomorphes permettant le rendu de vues React et de vues React-Router.
```

File: content/community/tools-misc.md [hunk 3]
```diff
@@ -1,19 +1,18 @@
...
+* **[react-render-visualizer](https://github.com/redsunsoft/react-render-visualizer)** : une façon visuelle de voir ce qui est (re)rendu et pourquoi.
+* **[javascript-monads](https://github.com/dschalk/javascript-monads)** : avec la maturité de ce projet, les monades deviennent plus que de simples jouets. React est utilisé de manière peu orthodoxe et si vous souhaitez utiliser React de la façon dont il est utilisé chez Facebook, alors ce n'est pas fait pour vous. Le code transpilé pour les trois classes de monades peut être téléchargé en exécutant `npm install reactive-monads`.
```
Comment (tdd) [inline content/community/tools-misc.md]: As this project matures, the monads are becoming more than mere toys. React is used in unorthodox ways, so if you want to use React the way they do at Facebook, this is not for you. Transpiled code for the three monad classes can be downloaded by entering "npm install reactive-monads".

```suggestion
* **[javascript-monads](https://github.com/dschalk/javascript-monads)** : alors que ce projet gagne en maturité, les monades deviennent plus que de simples jouets. React est utilisé de manière peu orthodoxe et si vous souhaitez utiliser React comme le fait Facebook, alors ce n'est pas fait pour vous. Le code transpilé pour les trois classes de monades peut être téléchargé en exécutant `npm install reactive-monads`.
```

File: content/community/tools-misc.md [hunk 4]
```diff
@@ -1,19 +1,18 @@
...
+* **[ReactCSS](http://reactcss.com/)** : styles en-ligne dans le JavaScript avec un support pour React.
```
Comment (tdd) [inline content/community/tools-misc.md]:
```suggestion
* **[ReactCSS](http://reactcss.com/)** : styles en-ligne dans le JavaScript avec une prise en charge de React.
```

File: content/community/tools-misc.md [hunk 5]
```diff
@@ -1,19 +1,18 @@
...
+* **[ReactQuestionnaire](https://github.com/kouryuu/react-questionnaire)** : composants React simples pour construire un questionnaire ou un sondage.
+* **[renderjs.io](https://renderjs.io/)** : SEO (*Search Engine Optimization* pour [Optimisation pour les moteurs de recherche](https://fr.wikipedia.org/wiki/Optimisation_pour_les_moteurs_de_recherche), NdT) pour React. Service pour assurer le référencement et le partage d'applications React.
```
Comment (tdd) [inline content/community/tools-misc.md]:
```suggestion
* **[renderjs.io](https://renderjs.io/)** : SEO *(Search Engine Optimization pour [Optimisation pour les moteurs de recherche](https://fr.wikipedia.org/wiki/Optimisation_pour_les_moteurs_de_recherche), NdT)* pour React. Service pour permettre aux applications React d'être analysées par les moteurs de recherche et partagées.
```

**Pull request merged**